test/e2e_node/cpu_manager_test.go

```diff
@@ -47,27 +49,130 @@ type ctnAttribute struct {
 	ctnName    string
 	cpuRequest string
 	cpuLimit   string
+	memRequest string
+	memLimit   string
+	command    string
+}
+
+func (ca ctnAttribute) Clone() ctnAttribute {
+	return ctnAttribute{
+		ctnName:    ca.ctnName,
+		cpuRequest: ca.cpuRequest,
+		cpuLimit:   ca.cpuLimit,
+		memRequest: ca.memRequest,
+		memLimit:   ca.memLimit,
+		command:    ca.command,
+	}
+}
+
+func (ca ctnAttribute) WithDefaults() ctnAttribute {
```

Comment:
the naming (or the functionality maybe?) could be a bit misleading if one don't check the implementation of this function, as I would have expected it to set default values to all the elements of the struct if they aren't initialized. unless the tests here assume that the cpus are always set.